### DISCUSSION TITLE
I just can't apply to the Github student program

### DISCUSSION THREAD

**Discussion by leonardop21** [community/community#64875] [GitHub Education]
### Hi everyone,

I just can't apply to the Github student program

I'm using a college email, where I do it remotely, there is no option to upload the documentation, I need to turn on the laptop's camera and take a photo to send and github still expects the image quality to be good? It is serious? And who doesn't have a super computer with a HD webcam?

When I manage to take a picture with good quality, github informs me that apparently I am not on the school campus and if I am using a vpn it is to deactivate.

Obviously I'm not on campus it's remote.

I've lost count how long I've been trying to sign up.

How to proceed?

**Comment by Shevilll** [comment]
I'm having the same issue.

**Comment by ghost** [comment]
I also have this problem, it seems that this problem has been around for a long time, I think this is a bug in github, but I don't know how to report the problem to the official, so that they can fix this problem as soon as possible #64847
